src/components/Navbar/index.jsx

```diff
@@ -1,17 +1,12 @@
 import Image from "next/image"
 import Link from "next/link"
 import { useState } from "react"
-import {
-  AiOutlineClose,
-  AiOutlineSearch,
-  AiOutlineShoppingCart,
-} from "react-icons/ai"
+import { AiOutlineClose, AiOutlineSearch } from "react-icons/ai"
+import { HiHeart } from "react-icons/hi"
 import { RxHamburgerMenu } from "react-icons/rx"
 import { TbArrowBadgeDown } from "react-icons/tb"
 import { TfiWorld } from "react-icons/tfi"
 
-import LogoImg from "../../../public/Logo.png"
-
 export default function Navbar() {
   const [languages, setLanguages] = useState(false)
   const [open, setOpen] = useState(false)
```

Comment:
Are these unused?

Reply:
no it used the first is for the languages slider the second is for the small screen navbar options